source/learn/simple-tableview.html.markdown

```diff
@@ -0,0 +1,178 @@
+In this tutorial we are creating a simple cell-based CTableView. CPTableView is one of the simplest way to organize our data into columns and rows. 
+Following this tutorial you will be able to understand how populate data into CPTableView, how you can add or remove elements.
+![](simpletableview/simpletableview1.png)
+
+We assume you already read and understood the [Xcode Interface Builder tutorial](xcode-basics.html).
+
+### Create the project
+
+First, go to the folder in which you want create the project. Run
+the `capp` command:
+
+    :::sh
+    capp gen -l -t NibApplication SimpleTableView
+
+This will generate the startup project using the Nib template.
+
+### Code
+
+Let’s define the required instance variables:
+
+	:::objj
+	@implementation AppController : CPObject
+	{
+	    @outlet CPWindow    theWindow;
+	    @outlet CPTableView tableView;
+	    CPMutableArray      nameArray;
+	}
+
+Our dataSource will be CPMutableArray:
+	
+	:::objj
+	- (id)init
+	{
+	    if (self = [super init]);
+	    {
+	        nameArray = [[CPMutableArray alloc]initWithObjects:"Alpha", "Beta", "Charlie", "Delta"];
```

Comment:
You have probably a style issue here, did you use capp_lint ? (space between ] and initWithObjects